### PR TITLE
Initial Saint Louis Event Pull

### PR DESCRIPTION
**Pull request by sblackmon**
This will be the first pull featuring information about the Saint Louis event in 2017.

### CODE REVIEW

**Comment by MikeRosTX** [comment]
@sblackmon I would reconsider and avoid redirecting propose on the navbar to Trello.  Hit me up on Slack if you wish to discuss it further.

File: data/events/2017-saintlouis.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: "2017-saintlouis" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
```
Comment (bridgetkromhout) [inline data/events/2017-saintlouis.yml]: This should be `2017-saint-louis`.

**Comment by bridgetkromhout** [comment]
Based on the Slack discussion, @sblackmon is considering/discussing their CFP process with his team.

Point of order: if you open your CFP (through whatever mechanism you prefer), list the CFP open/close/announce dates in your data file. That gets picked up automatically for https://www.devopsdays.org/speaking/

Also, you want `saint-louis` for everywhere you have `saintlouis` including the content directory name. It's a URL SEO thing. I'll investigate to see why the make_new_event script failed to do that.

**Comment by MikeRosTX** [comment]
Added **in progress** label as we're waiting on @sblackmon to discuss with team the use of Trello for Proposals before merging PR.

File: data/events/2017-saintlouis.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#
+coordinates: "38.7084, -90.4390" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Saint Louis" # Defaults to city, but you can make it the venue name.
+#
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: welcome
+#  - name: program
+  - name: propose
+    url: https://trello.com/b/qtP2nsVm/presentations # The url setting is optional, and only if you want the navigation to link off-site
```
Comment (mattstratton) [inline data/events/2017-saintlouis.yml]: I also don't recommend setting this until your CFP is open, as people *will* click on this link.

File: data/events/2017-saintlouis.yml (hunk 1 above)
Comment (mattstratton) [inline data/events/2017-saintlouis.yml]: Please update both the line and the filename; the filename should be `2017-saint-louis.yml`

**Comment by MikeRosTX** [comment]
Any updates @sblackmon?

**Comment by bridgetkromhout** [comment]
I'm going to merge and fix this.

**Pull request merged**